app/models/git/track.rb

```diff
@@ -51,22 +51,22 @@ def key_features
 
     memoize
     def has_concept_exercises?
-      !!config[:status][:concept_exercises]
+      !!status[:concept_exercises]
```

Comment:
I've changed this to use a (memoized) `status` method to gracefully handle tracks that don't have a `status` property.